app/uk/gov/hmrc/soletraderidentificationfrontend/models/JourneyLabels.scala

```diff
@@ -16,16 +16,41 @@
 
 package uk.gov.hmrc.soletraderidentificationfrontend.models
 
+import play.api.libs.functional.syntax._
 import play.api.libs.json.{JsPath, OFormat, OWrites, Reads}
 
-case class JourneyLabels(welsh: TranslationLabels)
+case class JourneyLabels(optWelshServiceName: Option[String],
+                         optEnglishServiceName: Option[String],
+                         optEnglishFullNamePageLabel: Option[String],
```

Comment:
optEnglishFullNamePageLabel should be last, otherwise the reads/writes below will not work correctly